src/main/java/com/redhat/lightblue/camel/LightblueComponent.java

```diff
@@ -28,11 +28,26 @@ public LightblueComponent(CamelContext context) {
     }
 
     @Override
+    /*
+     * @param uri the full URI of the endpoint
+     * 
+     * @param remaining the remaining part of the URI without the query parameters or component prefix, used to identify the lightblue
+     * request. since there is only one consumer right now, its okay to use remaining clause for identifying the request. if needed, we can
+     * use a uri param
+     * 
+     * @param parameters the optional parameters passed in
+     * 
+     * @return a newly created endpoint or null if the endpoint cannot be created based on the inputs
+     * 
+     * @throws Exception is thrown if error creating the endpoint
+     * 
+     * (non-Javadoc)
```

Comment:
@dcrissman eclipse added that, apparently it does so to support older versions of tools that parse javadocs. removed it.